.github/workflows/node.js.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 13.x, 14.x, 15.x]
+        node-version: [18.x, 19.x, 20.x, 21.x]
```

Comment:
I think we could omit non-LTS releases here. Or we can keep 21.x, because that might enable us to detect issues early that would also occur with 22.x when it comes out. But let's not test 19.x.